src/Hng.Web/Hng.Web.csproj

```diff
@@ -26,7 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\HNG-Boilerplate-Application\HNG-Boilerplate-Application.csproj" />
+    <!--<ProjectReference Include="..\HNG-Boilerplate-Application\HNG-Boilerplate-Application.csproj" />-->
```

Comment:
Why did you remove this reference?

Reply:
The csproj file seems to be missing, so i had to comment it out

Reply:
Ok